<launch>
  <arg name="control_config_file" default="$(find agv_container)/config/params_control.yaml"/>
  <node pkg="agv_container" type="ros2mqtt.py" name="ros2mqtt" output="screen">
    <rosparam file="$(arg control_config_file)" command="load"/>
  </node>
</launch>